### PR TITLE
Removed references of android-rs-glue from comments and documentation

### PR DESCRIPTION
**Pull request by saurvs**
Fixes https://github.com/servo/servo/issues/9507.

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9549)

### CODE REVIEW

File: python/servo/command_base.py [hunk 1]
```diff
@@ -362,11 +362,6 @@ def build_env(self, gonk=False, hosts_file_path=None):
             env["OPENSSL_LIB_DIR"] = openssl_dir
             env['OPENSSL_INCLUDE_DIR'] = path.join(env["GONKDIR"], "external/openssl/include")
 
-        # FIXME: These are set because they are the variable names that
```
Comment (KiChjang) [inline python/servo/command_base.py]: This comment is removed, but the code was not changed.

**Comment by saurvs** [comment]
Yeah, I was unsure about that because the issue said the references linger on only in comments and documentation, and I didn't want to break anything by removing code I wasn't fully familiar with. But I'll correct it now.

**Comment by Wafflespeanut** [comment]
Actually, never mind (didn't look at the issue)...

**Comment by larsbergstrom** [comment]
Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[python/servo/command_base.py, line 365 \[r2\]](https://reviewable.io:443/reviews/servo/servo/9549#-K9lsLqr_0n1MnQI5ZJn:-K9lsLqr_0n1MnQI5ZJo:16788457) ([raw file](https://github.com/servo/servo/blob/fb3fe3d784c24cdfd8040af0282e5c2182ee2051/python/servo/command_base.py#L365)):*
Please just change the comment to mention build-apk instead. Sorry I didn't make that clear! These variables are still used in the build scripts that were based on the original ones from android-rs-glue.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/servo/servo/9549)*

File: python/servo/command_base.py (hunk 1 above)
Comment (larsbergstrom) [inline python/servo/command_base.py]: Yeah, sorry I didn't see this PR come in! He does not need to change the code - just the comment. Sorry I wasn't clear!

**Comment by saurvs** [comment]
So should the original comment stay with 'android-rs-glue' replaced by 'build-apk'?

**Comment by larsbergstrom** [comment]
@saurvs Yes, please! Also, you can remove the `FIXME`, since I don't think it's going anywhere now :-)

**Comment by saurvs** [comment]
Ok!

**Comment by larsbergstrom** [comment]
Looks great! Can you please squash the commits, and then I'll land it? Let me know if you need help with the git commands to squash them :-)

Thanks for your patience!

**Comment by saurvs** [comment]
Squashing done!

**Comment by KiChjang** [comment]
@bors-servo retry
* #7791

**Pull request merged**